### ISSUE TITLE
remove "push notification" persistent notification

### ISSUE THREAD

**Issue by jeroen7s** [Telegram-FOSS-Team/Telegram-FOSS#372]
Another FOSS app on fdroid seems to manage push notifications fine without showing an anoying persistent notification all the time: Tutanota.

Not sure what they are doing differently, but they don't seem to need the persistent notification in order to have a working push system.
https://f-droid.org/en/2018/09/03/replacing-gcm-in-tutanota.html

**Comment by niansa** [comment]
@jeroen7s Tutanota simply sends out a notification every like 15 Minutes and removes it before anyone would notice it.

**Comment by jeroen7s** [comment]
Don't think that's the case, from the blog it seems like they check every 15 minutes if the service is still running that checks for push notifications, which is not the same.

And even if they are showing a notification every 15 minutes it seems to be working pretty seemlessly.

I'm really getting tired of persistent notifications for everything. I don't want a notification all the time, for: Telegram, Signal, Riot, Briar... It adds up.

**Comment by dic1911** [comment]
Just simply disable "Push Notification Service" notification channel. Problem solved.

**Comment by thermatk** [comment]
@jeroen7s No, it attacks you with notification from system about "Telegram using battery". If you hide this one too, you're good

**Comment by soredake** [comment]
@thermatk on android q you can disable notification without new notification about battery usage.
